types/doubleclick-gpt/index.d.ts

```diff
@@ -155,32 +164,23 @@ declare namespace googletag {
     }
 
     namespace events {
-        interface ImpressionViewableEvent {
-            serviceName: string;
-            slot: Slot;
-        }
-
-        interface SlotOnloadEvent {
```

Comment:
Removing these would be a breaking change to anyone relying on them. According to the [docs](googletageventsimpressionviewableevent) these simply extend `googletag.events.Event`... might be worth keeping them as empty extensions.

Reply:
I did initially have them as empty extensions but tslint complained about them. I’ll re add and address the linter by disabling the rule. Good call.

Reply:
Ah, yes. You can either turn off the linter there, or create "types":

```typescript
type SlotOnloadEvent = Event;
```